docs/reproducing-results.md

```diff
@@ -362,9 +362,59 @@ run inference through Nvidia NIM API.
        subprocess.run(cmd, shell=True, check=True)
    ```
 
-6. Now all the data is generated and you can follow up by converting it to the SFT format.
-
-   TBD
+6. Check for test set contamination.
+   We test against GSM8K, MATH, AMC 2023, and AIME 2024.  
+
+   First we use this command to get all the unique problems:
+   ```
+   cat /workspace/new-problems-solution-augmentation/*/*/output-rs0.jsonl | jq '.problem' | sort -u | shuf | jq -c -R '{"problem": .}' > /workspace/new-problems-solution-augmentation/unique_problems_all.jsonl
```

Comment:
This shouldn't be needed, I think - the retrieve_similar already builds a set, so it will only operate on unique problems

Reply:
Cool!